firebase-app-distribution/src/test/java/com/google/firebase/app/distribution/FirebaseAppDistributionTest.java

```diff
@@ -475,4 +475,20 @@ public void updateApp_withApkReleaseAvailable_returnsSameApkTask() {
 
     assertEquals(updateTask, updateTaskToReturn);
   }
+
+  private UpdateTask updateToNewRelease(AppDistributionReleaseInternal newRelease) {
+    when(mockSignInStorage.getSignInStatus()).thenReturn(true);
+    when(mockCheckForNewReleaseClient.checkForNewRelease()).thenReturn(Tasks.forResult(newRelease));
+    firebaseAppDistribution.setCachedNewRelease(newRelease);
```

Comment:
Why do we have to both mock checkForNewRelease() AND set the cached new release?

Reply:
We don't. Good catch. (I think this is dead code from a previous implementation)